### PR TITLE
Roshan gpg key

### PR DESCRIPTION
**Pull request by rbhrzlabs**

### CODE REVIEW

**Review by ysibirski — DISMISSED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -234,5 +234,11 @@ <h2>Alberto Sala @alsala</h2>
     <a href="https://keys.openpgp.org/vks/v1/by-fingerprint/9C56E9D0C4B47AE266466B68026DF8A70A1548AB">0x026DF8A70A1548AB</a>
     <h4>Key fingerprint=9C56 E9D0 C4B4 7AE2 6646 6B68 026D F8A7 0A15 48AB</h4>
     <hr/>
+   
+    <h2>Roshan  @rbhrzlabs</h2>
+    <a href="keys/rbhave.asc"><rbhave.asc</a>
+    <a href="https://keys.openpgp.org/vks/v1/by-fingerprint/56020240F2BDDE011C10F1A08123F4B2EC40E890">0x8123F4B2EC40E890</a>
```
Comment (ysibirski) [inline index.html:240]: Link is pointing to the WRONG fingerprint

**Review by ysibirski — CHANGES_REQUESTED** [review]
Found 1 issue

**Review by rbhrzlabs — COMMENTED** [review]
fixed the index.html with correct URL

**Review by ysibirski — APPROVED** [review]
LGTM

File: index.html [hunk 2]
```diff
@@ -237,7 +237,11 @@ <h4>Key fingerprint=9C56 E9D0 C4B4 7AE2 6646 6B68 026D F8A7 0A15 48AB</h4>
    
     <h2>Roshan  @rbhrzlabs</h2>
     <a href="keys/rbhave.asc"><rbhave.asc</a>
+<<<<<<< HEAD
```
Comment (ysibirski) [inline index.html:240]: Merge conflict lines made into the commit. It needs to be resolved

**Pull request closed without merging**